mmv1/third_party/terraform/services/bigquerydatatransfer/resource_bigquery_data_transfer_config_test.go

```diff
@@ -383,6 +383,29 @@ func testAccBigqueryDataTransferConfig_scheduledQuery_update(t *testing.T) {
 	})
 }
 
+func testAccBigqueryDataTransferConfig_CMEK(t *testing.T) {
```

Comment:
Any reason to skip the VCR run? Also I think this function should be named `TestAccBigqueryDataTransferConfig_CMEK`

Reply:
same reason from above https://github.com/GoogleCloudPlatform/magic-modules/pull/11478#discussion_r1729437326

I also agree with the naming convention but went with what i committed due to the whole test file matching it.

Should we do the renaming of all the tests here or in a separate PR?

Reply:
Ah thanks for letting me know. If other tests are created in this way then I'm fine with it.